plugins/cue_points/base/batch/copyCuePoints/Engine/KCopyCuePointEngine.php

```diff
@@ -76,7 +77,12 @@ protected function copyCuePointsToEntry($srcEntryId, $destEntryId)
 
 	protected function copySingleCuePoint($cuePoint, $destEntryId)
 	{
-		$clonedCuePoint = KBatchBase::tryExecuteApiCall(array('KCopyCuePointEngine','cuePointClone'), array($cuePoint->id, $destEntryId));
+		$parentId = isset($this->idsMap[$cuePoint->parentId]) ? $this->idsMap[$cuePoint->parentId] : null;
```

Comment:
parentId = null;
if($cuePoint->parentId)
{
	if(isset($this->idsMap[$cuePoint->parentId]))
	{
		parentId = $this->idsMap[$cuePoint->parentId];
	}
	else
	{
		warninig: Cuepoint ID , Partner ID - does not exist in Map.
	}